include/picongpu/versionFormat.cpp

```diff
@@ -138,6 +141,20 @@ namespace picongpu
         adios << versionNotFound;
 #endif
 
+        std::stringstream openPMD;
+#if( ENABLE_OPENPMD == 1 )
+        openPMD << OPENPMDAPI_VERSION_MAJOR
+                << "."
+                << OPENPMDAPI_VERSION_MINOR
+                << "."
+                << OPENPMDAPI_VERSION_PATCH
+                << " ("
```

Comment:
note: `OPENPMDAPI_VERSION_LABEL` might be empty and should only be added if it's a non-empty string.
Examples:
- 0.6.2-alpha
- 1.2.3-rc1
- 1.2.3

Reply:
I changed that part, but is the value actually used anywhere? Everywhere throughout the codebase where `getSoftwareVersions( std::ostream & cliText )` is called, the return value is discarded. Only the writes to `cliText` have an effect, really.

Reply:
It's still a bit off: it must be `-label` in semver (see example in first comment).

The idea is to include this in openPMD meta info at some point https://github.com/openPMD/openPMD-standard/issues/137 as well as `picongpu --version`

Reply:
@franzpoeschel you can now use the new public API to build your version strings:
https://github.com/openPMD/openPMD-api/blob/0.11.0-alpha/include/openPMD/version.hpp#L54-L80

Reply:
I've changed it to use that now